subscription-service/src/test/kotlin/com/egm/stellio/subscription/service/SubscriptionServiceTests.kt

```diff
@@ -67,8 +67,20 @@ class SubscriptionServiceTests : TimescaleBasedTests() {
     @BeforeAll
     fun bootstrapSubscriptions() {
```

Comment:
Out of this PR scope, i think that this function needs a refactoring

Reply:
any tip ?

Reply:
With just one createSubscription() function that a subscription as argument and return the created subscription id

Reply:
ok